app/lib/feature_flags.rb

```diff
@@ -1,6 +1,8 @@
 class FeatureFlags
   ENABLED_AFTER_DATE = {
     example_2125_flag: { from: "2125-01-01", public: false },
+    thresholds_2025: { from: "2025-03-18", public: false },
```

Comment:
this line is added for UAT it will be removed and the line below will be uncommented priuor to merge